src/components/Blinds.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useState } from 'react';
+
+const Blinds = ({ state }) => {
+    const [open, setOpen] = useState(state);
+
+    return(
+        <div className='blinds'>
+            <div className="blinds__icon">
+                <img src={`./assets/blinds-${open}.svg`} />
```

Comment:
pro tip: takhle skládat názvy jde, ale ve větší aplikaci chceš mít názvy souborů vcelku a to jen čistě proto, abys dokázala najít jestli se nějaké obrázky používají. Představ si aplikaci s 300 obrázky a 600 soubory a budeš chtít zjistit jestli obrázek assets/blinds-open.svg se používá. Normálně stačí najít jestli někde není název "blinds-open.svg" ale pokud to takhle skládáš z hodnot proměnných, tak to nenajdeš. Tj já bych preferoval něco jako:
```
                <img src={open === "open" ?  `./assets/blinds-open.svg` : "./assets/blinds-closed.svg"} />
```